README.md

```diff
@@ -12,17 +12,16 @@ By default, `kcp` only knows about:
 - [`CustomResourceDefinition`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)s, to define new types
 - a handful of other low-level resources like [`Lease`](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/lease-v1/)s, [`Event`](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-application-introspection/)s, etc.
 
-![kubectl api-resources showing minimal API resources](./docs/images/kubectl-api-resources.png)
+![kubectl api-resources showing kcp's API resources](./docs/images/kubectl-api-resources.png)
 
-Like vanilla Kubernetes, `kcp` persists these resources in etcd for durable storage.
+Any other resources, including standard Kubernetes resources like [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)s and the rest, can be added as CRDs and optionally reconciled using the standard controllers launched against the `kcp` API, or via two-way replication into a Kubernetes cluster.
```

Comment:
"Two-way replication" may not be clear enough.

> ... can be added as CRDs and be replicated onto one or more Kubernetes clusters.

Reply:
#554